.cypress/cypress/integration/buckinghamshire.js

```diff
@@ -54,6 +54,43 @@ describe('buckinghamshire cobrand', function() {
     cy.contains('The road you have selected is on a regular gritting route').should('be.visible');
   });
 
+  describe("Parish grass cutting category speed limit question", function() {
+    it('displays the parish name if answer is "no"', function() {
+      cy.get('#map_box').click(290, 307);
+      cy.wait('@report-ajax');
+      cy.pickCategory('Grass cutting');
+      cy.wait('@around-ajax');
+      cy.nextPageReporting();
+      cy.get('#form_speed_limit_greater_than_30').select('no');
+      cy.nextPageReporting();
+      cy.nextPageReporting();
+      cy.contains('sent to Adstock Parish Council and also published online').should('be.visible');
+    });
+
+    it('displays the council name if answer is "yes"', function() {
+      cy.get('#map_box').click(290, 307);
+      cy.wait('@report-ajax');
+      cy.pickCategory('Grass cutting');
+      cy.wait('@around-ajax');
+      cy.nextPageReporting();
+      cy.get('#form_speed_limit_greater_than_30').select('yes');
+      cy.nextPageReporting();
+      cy.nextPageReporting();
+      cy.contains('sent to Buckinghamshire Council and also published online').should('be.visible');
+    });
+
+    it('displays the council name if answer is "dont_know"', function() {
+      cy.get('#map_box').click(290, 307);
+      cy.wait('@report-ajax');
+      cy.pickCategory('Grass cutting');
+      cy.wait('@around-ajax');
+      cy.nextPageReporting();
+      cy.get('#form_speed_limit_greater_than_30').select('dont_know');
+      cy.nextPageReporting();
+      cy.nextPageReporting();
+      cy.contains('sent to Buckinghamshire Council and also published online').should('be.visible');
+    });
+  });
```

Comment:
Thinking about it I should probably tidy up the duplication in this a bit using `beforeEach`, will do that quickly now.

Reply:
Done in 4e1c267682d2b8624b24156f7a56be748134f8fa